messaging/tck/src/main/java/org/eclipse/microprofile/reactive/messaging/tck/CompletionStageIncomingMethodVerification.java

```diff
@@ -68,8 +67,25 @@
     CompletionStageIncomingMethodVerification.OUTGOING_WRAPPED,
     CompletionStageIncomingMethodVerification.INCOMING_OUTGOING_WRAPPED
 })
+@Test(groups = {"all", "incoming"})
```

Comment:
Decided to use TestNG groups rather than controlling which tests are run via an injected component.